app/constants/sample.ts

```diff
@@ -0,0 +1,23 @@
+const sampleMembers = [
```

Comment:
@Suresh-ashwini Remove this file ..If not needed ..

Reply:
removed unused contant file

Reply:
Unused file removed